dipy/segment/clustering.py

```diff
@@ -416,7 +417,7 @@ class QuickBundles(Clustering):
     metric : str or `Metric` object (optional)
         The distance metric to use when comparing two streamlines. By default,
         the Minimum average Direct-Flip (MDF) distance [Garyfallidis12]_ is
-        used and requires streamlines to have the same number of points.
+        used and streamlines are automatically resampled so they have 18 points.
```

Comment:
It wouldn't be hard to implement. However, I don't know if this would make sense to add a parameter like that in the `QuickBundles` constructor since `QuickBundles` can accept any kind of metric that doesn't necessarily need a `nb_points`.

I would argue that users who just want to use the QuickBundles algorithm as a black box (I'm guessing it would be the majority) won't care about changing the number of points. This parameter has a direct impact on the speed of the  algorithm but also the quality. I'm afraid uninformed users would simply decrease that number to gain speed without necessarily considering the validity of the results.

An informed user can "easily" and explicitly change that parameter as follows
```python
feature = ResampleFeature(nb_points=42)
metric = AveragePointwiseEuclideanMetric(feature)
qb = QuickBundles(threshold=10., metric=metric)
clusters = qb.cluster(streamlines)
```
and I should probably add a note about that in the tutorial. What do you think? @Garyfallidis ?